coco/master.py

```diff
@@ -20,11 +22,18 @@
 from comet import Manager, CometError
 
 from . import Endpoint, SlackExporter, worker, __version__, RequestForwarder, State
+from .scheduler import Scheduler
 
 app = Sanic(__name__)
 app.config.update({"REDIS": {"address": ("127.0.0.1", 6379)}})
 redis = SanicRedis(app)
-logger = logging.getLogger(__name__)
+
+logger = logging.getLogger()
```

Comment:
What do these changes do?
Why did you remove the `__name__`?
Does the format apply to logging in all coco modules?
I think the log level is set in the config. So shouldn't be hardcoded here.

Reply:
I think removing the name ensures that this is getting the root logger. For some reason with `__name__` I wasn't getting any logs from calls in other modules. I thought these should be the same, put apparently not...

The format applies to all logging.

I can get it to set the logging level from the config.

Reply:
I thought that was working already

Reply:
with `__name__` I get no logs from the scheduler but without it I do.

Reply:
I guess you have to get the logger after starting the thread

Reply:
ok, in the end what was needed was to set the log level in the scheduler. I also moved the formatting and into `__init__.py` so that all modules inherit it.